wtxmgr/tx.go

```diff
@@ -805,6 +805,95 @@ func (s *Store) rollback(ns walletdb.ReadWriteBucket, height int32) error {
 	return putMinedBalance(ns, minedBalance)
 }
 
+// OutputsToWatch returns a list of outputs to monitor during the wallet's
+// startup. The returned items are similar to UnspentOutputs, exccept the
+// locked outputs and unmined credits are also returned here. In addition, we
+// only set the field `OutPoint` and `PkScript` for the `Credit`, as these are
+// the only fields used during the rescan.
+func (s *Store) OutputsToWatch(ns walletdb.ReadBucket) ([]Credit, error) {
+	var (
+		unspent []Credit
+		op      wire.OutPoint
+		block   Block
+	)
+
+	err := ns.NestedReadBucket(bucketUnspent).ForEach(func(k,
+		v []byte) error {
+
+		err := readCanonicalOutPoint(k, &op)
+		if err != nil {
+			return err
+		}
+
+		err = readUnspentBlock(v, &block)
+		if err != nil {
+			return err
+		}
+
+		// TODO(jrick): reading the entire transaction should be
```

Comment:
?

Reply:
Think this very old TODO is still relevant so wanna keep it.